blueocean-events/src/main/java/io/jenkins/blueocean/events/BlueMessageEnricher.java

```diff
@@ -59,22 +59,27 @@ public void enrich(@Nonnull Message message) {
         message.set(EventProps.Jenkins.jenkins_org, OrganizationImpl.INSTANCE.getName());
 
         String channelName = message.getChannelName();
-        if (channelName.equals(Events.JobChannel.NAME)) {
-            JobChannelMessage jobChannelMessage = (JobChannelMessage) message;
-            ParameterizedJobMixIn.ParameterizedJob job = jobChannelMessage.getJob();
-            Link jobUrl = LinkResolver.resolveLink(job);
 
-            jobChannelMessage.set(BlueEventProps.blueocean_job_rest_url, jobUrl.getHref());
-            jobChannelMessage.set(BlueEventProps.blueocean_job_pipeline_name, job.getFullName());
-            if (job instanceof WorkflowJob) {
-                ItemGroup<? extends Item> parent = job.getParent();
-                if (parent instanceof WorkflowMultiBranchProject) {
-                    String multiBranchProjectName = parent.getFullName();
-                    jobChannelMessage.set(EventProps.Job.job_ismultibranch, "true");
-                    jobChannelMessage.set(BlueEventProps.blueocean_job_pipeline_name, multiBranchProjectName);
-                    jobChannelMessage.set(BlueEventProps.blueocean_job_branch_name, job.getName());
-                }
-            }
+        if (!channelName.equals(Events.JobChannel.NAME)) {
+            return;
+        }
+
+        JobChannelMessage jobChannelMessage = (JobChannelMessage) message;
+        ParameterizedJobMixIn.ParameterizedJob job = jobChannelMessage.getJob();
+        Link jobUrl = LinkResolver.resolveLink(job);
+        if (jobUrl == null) {
```

Comment:
@i386 I'm not sure if this is the fix we want. For sure the null check is good, but it could be hiding issues. If a job exists we should be able to resolve the think afaik. The question I have is what job cant resulve a link? And how can we report issues like this in the future as it will affect the frontend sse handlers